packages/mwp-api-state/src/index.js

```diff
@@ -27,11 +27,11 @@ export { api, app, DEFAULT_API_STATE } from './reducer';
  * order to render the application. We may want to write a server-specific
  * middleware that doesn't include the other epics if performance is an issue
  */
-export const getApiMiddleware = (routes, fetchQueriesFn, baseUrl) =>
+export const getApiMiddleware = (resolveRoutes, fetchQueriesFn) =>
```

Comment:
this summarizes the effect of the refactor - instead of `routes` + `baseUrl`, the function just gets a `resolveRoutes` function